metricbeat/module/vsphere/vendor/vendor.json

```diff
@@ -0,0 +1,97 @@
+{
+	"comment": "",
+	"ignore": "test",
```

Comment:
I would expect here `"ignore": "test github.com/elastic/beats"` as otherwise if you remove the libs from the vendor dir and then run `govendor sync` it will also load libbeat etc. into it.